src/System.Private.ServiceModel/tools/test/BridgeCertificateInstaller/project.json

```diff
@@ -6,7 +6,9 @@
     "System.IO.FileSystem": "4.0.0",
     "System.IO": "4.0.10",
     "System.Net.Http": "4.1.0-rc3-24109-00",
-    "System.Net.Primitives": "4.0.10"
+    "System.Net.Primitives": "4.0.10",
+    "System.ServiceModel.Http": "4.1.0-rc3-24109-00",
+    "System.ServiceModel.Primitives": "4.1.0-rc3-24109-00"
```

Comment:
Ah I missed that you're using a WCF proxy here. 

Does BridgeCertificateInstaller require the latest S.SM.Primitives? If not, no need to bump it up to a version that's this high